linera-base/src/crypto.rs

```diff
@@ -321,40 +321,40 @@ impl From<CryptoHash> for [u64; 4] {
     }
 }
 
-impl std::fmt::Display for Signature {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::result::Result<(), std::fmt::Error> {
+impl fmt::Display for Signature {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> Result<(), fmt::Error> {
```

Comment:
Nit: maybe return `fmt::Result` here?

```suggestion
    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
```